src/main/java/edu/cmu/sei/kalki/db/daos/UmboxLookupDAO.java

```diff
@@ -69,9 +69,11 @@ public static UmboxLookup findUmboxLookup(int id) {
      * Finds all umbox lookups based on the given device id
      */
     public static List<UmboxLookup> findUmboxLookupsByDevice(int deviceId) {
-        String query = "SELECT ul.* FROM umbox_lookup ul, device d, device_type dt " +
+        /*String query = "SELECT ul.* FROM umbox_lookup ul, device d, device_type dt " +
```

Comment:
This whole change seems wrong. The function used to get umbox lookups by device. Now it was changed to find umbox lookups by device type, yet it still has the same name as before, and the functionality to get umbox lookups by device is now gone. Instead of modifying this method, a new method should be created, findUmboxLookupsByDeviceType(int deviceTypeId), and the old method should remain as it was.